src/zjs_aio_a101.c

```diff
@@ -142,13 +142,15 @@ static void ipm_msg_receive_callback(void *context, u32_t id,
 #ifdef DEBUG_BUILD
         u32_t pin = msg->data.aio.pin;
 #endif
+        ZVAL num = jerry_create_number(pin_value);
 
         switch (msg->type) {
         case TYPE_AIO_PIN_READ:
+            DBG_PRINT("aio async read %d\n", pin_value);
+            break;
```

Comment:
Read is going to do nothing now?

Reply:
This type of read msg from ARC is no longer needed for async reads, for simplicity of the API, both read() and readAsync() does a synchronous read over IPM, and asyncRead just call signal callback after it gets the result, so this type of message should never show up.